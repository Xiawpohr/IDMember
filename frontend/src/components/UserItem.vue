<template>
  <p>{{user.name}}</p>
</template>

<script>
export default {
  props: {
    user: {
      type: Object,
      required: true,
      default: () => ({})
    }
  }
}
</script>
