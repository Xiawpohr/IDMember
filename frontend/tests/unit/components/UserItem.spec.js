import { shallowMount } from '@vue/test-utils'
import UserItem from '@/components/UserItem.vue'

const user = { id: 1, name: 'Arthur' }

it('should render the component with user prop', () => {
  const userItem = shallowMount(UserItem, {
    propsData: {
      user
    }
  })
  expect(userItem.html()).toMatchSnapshot()
})
