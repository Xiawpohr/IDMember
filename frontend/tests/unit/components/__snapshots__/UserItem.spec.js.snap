// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component with user prop 1`] = `<p>Arthur</p>`;
